client/src/components/questions/Answer.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import API_KEY from '../../../../config.js';
+
+const StyledAnswer = styled.div`
+  padding: 0.7% 0;
+`;
+
+
+const dateParser = (rawDate) => {
+  var date = new Date(rawDate.replace(/-/g, '/').replace(/T.+/, ''));
+  var dateStr = date.toDateString();
+
+  dateStr = dateStr.slice(dateStr.indexOf(' '));
+
+  var secondSpaceIndex = dateStr.indexOf(' ', 5);
+
+  dateStr = dateStr.slice(0, secondSpaceIndex) + ',' + dateStr.slice(secondSpaceIndex);
+
+  return dateStr;
+};
+
+const Answer = ({ answerData }) => {
+
+  const [reportText, setReportText] = useState(<u>Report</u>);
+
+  const [checkHelpfulClick, setCheckHelpfulClick] = useState(false);
+  const [helpfulCount, setHelpfulCount] = useState(answerData.helpfulness);
+
+  const dateStr = dateParser(answerData.date);
+  let user = answerData.answerer_name;
+
+  if (answerData.answerer_name === 'Seller') {
+    user = <strong>Seller</strong>;
+  }
+
+  var helpfulClick = () => {
+    if (checkHelpfulClick === true) {
+      return;
+    } else {
+      setHelpfulCount(helpfulCount + 1);
+      axios({
+        method: 'put',
+        url:`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/qa/answers/${answerData.id}/helpful`,
+        headers: {Authorization: API_KEY}
+      })
+      .then((response) => {
+        setCheckHelpfulClick(true);
+      })
+      .catch((err) => {
+        //console.log(err);
+      })
+    }
+  }
```

Comment:
Is the if statement on line 41 just making sure that the function won't be able to send more than one helpful put request?

Reply:
yes, precisely.